sql/teamwork_database_1.1.sql

```diff
@@ -0,0 +1,134 @@
+-- 用户表
+SET FOREIGN_KEY_CHECKS = 0; -- 禁用外键约束，创建完后恢复
+DROP TABLE IF EXISTS tbl_user;
+CREATE TABLE tbl_user (
+  uid         INT PRIMARY KEY,
+  account     VARCHAR(255) UNIQUE NOT NULL,
+  email       CHAR(50) UNIQUE     NOT NULL,
+  password    CHAR(50)            NOT NULL,
+  image       VARCHAR(255),
+  create_time DATETIME
+
+)
+  ENGINE = InnoDB DEFAULT CHARSET = utf8;
+
+DELIMITER |
+CREATE TRIGGER trigger_user_create BEFORE INSERT ON tbl_user
+FOR EACH ROW
+  IF new.create_time IS NULL
+  THEN
+    SET new.create_time = now();
+  END IF;
+|
+DELIMITER ;
+
+-- 工程表
+DROP TABLE IF EXISTS tbl_project;
+CREATE TABLE tbl_project (
+  pid         INT PRIMARY KEY,
+  name        VARCHAR(255) UNIQUE NOT NULL,
+  uid         INT                 NOT NULL, -- 文档中的owner，我认为名字和依赖表一致方便查询
+  description VARCHAR(255)        NOT NULL,
+  image       VARCHAR(255),
+  create_time DATETIME,
+  FOREIGN KEY (uid) REFERENCES tbl_user (uid) ON DELETE CASCADE ON UPDATE CASCADE
```

Comment:
Constraints on foreign key are not an efficient way. Remove it and guarantee the constraints in code